### DISCUSSION TITLE
can't find & install omnik inverter over integration home assistant

### DISCUSSION THREAD

**Discussion by ap20132022** [robbinjanssen/home-assistant-omnik-inverter#147] [Q&A]
Hi everyone,
I'm new here. I tried to install this integration with home assistant. I go to "Setting" --> "Device & Sevices" and click on "add integration". But I can't find omnik inverter.
Can anyone help me?
Thanks a lot and have a nice weekend.

**Comment by ap20132022** [comment]
after a while (3 hours), I installed it manually. It works now. It can be closed now.

**Comment by klaasnicolaas** [comment]
Did you install it via HACS and then restart HA first?

**Comment by ap20132022** [comment]
I downloaded the files, copied it to the folder /config/....
At last, I restarted HA
